lib/matplotlib/colorbar.py

```diff
@@ -1487,6 +1489,15 @@ def make_axes_gridspec(parent, *, location=None, orientation=None,
     fig = parent.get_figure()
     cax = fig.add_subplot(ss_cb, label="<colorbar>")
     cax.set_aspect(aspect, anchor=loc_settings["anchor"], adjustable='box')
+    cax._colorbar_info = dict(
+        location=location,
+        parents=[parent],
+        shrink=shrink,
+        anchor=anchor,
+        panchor=panchor,
```

Comment:
This doesn't appear to be read anywhere? Is that a bug elsewhere?

Reply:
No the point is just to be consistent with the non-gridspec axes.  In particular we can now check for the attribute.  However, after I put this in I realized that we can check if an axes is a colorbar more simply just by whether it is a ColorbarAxes, so maybe this isn't really needed.

Reply:
Well, I only meant `panchor`, but if you think there's a better way to do this entirely..

Reply:
Oh I see.  I included panchor just as a precaution - we may want to handle it better in the future.